rtdata/languages/default

```diff
@@ -2904,6 +2906,13 @@ TP_LOCALLAB_GAMM;Gamma
 TP_LOCALLAB_GAMMASKCOL;Gamma
 TP_LOCALLAB_GAMMASK_TOOLTIP;Adjusting Gamma and Slope can provide a soft and artifact-free transformation of the mask by progressively modifying 'L' to avoid any discontinuities.
 TP_LOCALLAB_GAMSH;Gamma
+TP_LOCALLAB_GAMUT_TOOLTIP;Default : Munsell.\nMunsell: fixes Lab mode hue drifts due to non-linearity, when chromaticity is changed.\nLab: applies a gamut control, in relative colorimetric, Munsell is then applied.\nXYZ Absolute, applies gamut control, in absolute colorimetric, Munsell is then applied.\nXYZ Relative, applies gamut control, in relative colorimetric, Munsell is then applied.
+TP_LOCALLAB_GAMUTTYPE;Avoir Color Shift
```

Comment:
We can reuse TP_LABCURVE_AVOIDCOLORSHIFT and TP_LOCALLAB_AVOID

Reply:
Done

Reply:
done...But my english ???